Login Screen/Sources/App/SceneDelegate.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  SceneDelegate.swift
+//  Login Screen
+//
+//  Created by Serhii  on 19/08/2022.
+//
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+
+        guard let windowScene = (scene as? UIWindowScene) else { return }
+        let window = UIWindow(windowScene: windowScene)
+        window.rootViewController = LoginViewController()
+        window.makeKeyAndVisible()
+        self.window = window
```

Comment:
Окно у нас уже есть просто его нужно проинициализировать
```suggestion
        window = UIWindow(windowScene: windowScene)
        window.rootViewController = LoginViewController()
        window.makeKeyAndVisible()
```